main/modes/flight/flight.c

```diff
@@ -762,6 +762,7 @@ static void flightUpdate(void* arg __attribute__((unused)), int64_t elapsedUs)
         case FLIGHT_MENU:
         {
             drawMenuLogbook(flight->menu, flight->menuLogbookRenderer, elapsedUs);
+            drawText(&flight->logbook, 206, "rB", 237, 1823);
```

Comment:
What is this line? It seems like it draws two chars waaaay offscreen. Also `206` shouldn't be used as a color value. The equivalent color is `c542`
```c
int16_t drawText(const font_t* font, paletteColor_t color, const char* text, int16_t xOff, int16_t yOff)
```